source/cellular_3gpp_urc_handler.c

```diff
@@ -580,6 +740,29 @@ CellularPktStatus_t _Cellular_ParseRegStatus( CellularContext_t * pContext,
             atCoreStatus = Cellular_ATGetNextTok( &pRegStr, &pToken );
         }
 
+        if( atCoreStatus == CELLULAR_AT_SUCCESS )
```

Comment:
Would like to suggest try to avoid adding new function pointer type if it is possible.

Reply:
As discussed, adding function pointer in .c file is harmless.